apiary.apib

```diff
@@ -361,6 +361,36 @@ We consider the following changes to be backwards-incompatible:
 - Changing the format of a parameter of API responses (eg. integer to float)
 - Making validation of a parameter more strict (eg. making it required)
 - Returning a different HTTP status code.
+
+<a name="additions"></a>
+### Additions
+
+We list all backwards-compatible additions here. These are currently available in all published versions.
+(There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
+
+#### Latest
+
+- In `users.list`, you can now also request deactivated users, by using a new filter on `status`.  
```

Comment:
There are a lot of commas in these sentences which is a dutch thing. Would run this text through Grammarly and see which ones can be removed

Reply:
I did now... It doesn't give any recommendations... so keeping it as it is...